desktop/renderer/index.js

```diff
@@ -41,6 +42,16 @@ class Keybase extends Component {
   constructor () {
     super()
 
+    // This net.connect() is a heinous hack.
+    //
+    // On Windows, but *only* in the renderer thread, our RPC connection
+    // hangs until other random net module operations, at which point it
+    // unblocks.  Could be Electron, could be a node-framed-msgpack-rpc
+    // bug, who knows.
+    if (process.platform === 'win32') {
```

Comment:
if we do need to keep this hack maybe let's just isolate it in a different module and not in here. So we can do like
```
import windowsHack from '../../react-native/react/native/windows-hack'
```
then just windowsHack()